sklearn/model_selection/tests/test_search.py

```diff
@@ -245,76 +246,36 @@ def test_grid_search_labels():
         gs.fit(X, y)
 
 
-def test_trivial_grid_scores():
-    # Test search over a "grid" with only one point.
-    # Non-regression test: grid_scores_ wouldn't be set by GridSearchCV.
+def test_trivial_results_attr():
+    # Test search over a "grid" with only one point. (NRT for gh-1782)
```

Comment:
I don't think you should introduce this shorthand. There is ample space for a more accessible comment.